index.html

```diff
@@ -98,6 +103,13 @@ <h2>Testing port connectivity:</h2>
     "port": 8080,
     "reachable": false
 }</pre>
+{{ end }}
+      </div>
+{{ if .IsLookupCityEnabled }}
```

Comment:
Should check that both city and country are not "Unknown".

Reply:
Not sure the way I implemented it is the "right" way in `Go` terms. I decided to check both for having the Country/City Lookup enabled _and_ to check whether any of the two values returns `"Unknown"`. I didn't find another way to properly chain the checks, hope that's fine.